src/clientversion.h

```diff
@@ -16,7 +16,7 @@
 //! These need to be macros, as clientversion.cpp's and bitcoin*-res.rc's voodoo requires it
 #define CLIENT_VERSION_MAJOR 0
 #define CLIENT_VERSION_MINOR 11
-#define CLIENT_VERSION_REVISION 2
+#define CLIENT_VERSION_REVISION 3
```

Comment:
NACK. This is the minor revision number ``N`` in ``v0.11.<N>.z3``. I don't think the ``.z3`` is embodied anywhere in this file, so let's remove mention of this file in ``release-process.md``.